### PR TITLE
Add more Aspnetcore namespace docs

### PR DESCRIPTION
**Pull request by wtgodbe**
Addresses the remaining items in https://github.com/dotnet/AspNetApiDocs/compare/wtgodbe/Docs?expand=1

### CODE REVIEW

File: aspnet-core/xml/ns-Microsoft.AspNetCore.Cors.Infrastructure.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Cors.Infrastructure">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Contains types that are used to create, configure, and handle CORS services</summary>
+    <remarks>For more information, see <see href="https://docs.microsoft.com/en-us/aspnet/core/security/cors">CORS</see>.</remarks>
```
Comment (JamesNK) [inline aspnet-core/xml/ns-Microsoft.AspNetCore.Cors.Infrastructure.xml:4]:
```suggestion
    <remarks>For more information, see <see href="https://docs.microsoft.com/aspnet/core/security/cors">CORS</see>.</remarks>
```

File: aspnet-core/xml/ns-Microsoft.AspNetCore.Cors.Infrastructure.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Cors.Infrastructure">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Contains types that are used to create, configure, and handle CORS services</summary>
```
Comment (JamesNK) [inline aspnet-core/xml/ns-Microsoft.AspNetCore.Cors.Infrastructure.xml:3]:
```suggestion
    <summary>Contains types that are used to create, configure, and handle CORS services.</summary>
```

File: aspnet-core/xml/ns-Microsoft.AspNetCore.Cors.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Cors">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Contains types that are used to enable cross-origin resource sharing.</summary>
+    <remarks>For more information, see <see href="https://docs.microsoft.com/en-us/aspnet/core/security/cors#enable-cors-with-attributes">Enable CORS with attributes</see>.</remarks>
```
Comment (JamesNK) [inline aspnet-core/xml/ns-Microsoft.AspNetCore.Cors.xml:4]:
```suggestion
    <remarks>For more information, see <see href="https://docs.microsoft.com/aspnet/core/security/cors#enable-cors-with-attributes">Enable CORS with attributes</see>.</remarks>
```

File: aspnet-core/xml/ns-Microsoft.AspNetCore.Cors.xml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Cors">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Contains types that are used to enable cross-origin resource sharing.</summary>
```
Comment (JamesNK) [inline aspnet-core/xml/ns-Microsoft.AspNetCore.Cors.xml:3]:
```suggestion
    <summary>Contains types that are used to enable cross-origin resource sharing (CORS).</summary>
```

File: aspnet-core/xml/ns-Microsoft.AspNetCore.Http.Headers.xml [hunk 5]
```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Http.Headers">
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Contains types for creating headers for HTTP requests and responses.</summary>
```
Comment (JamesNK) [inline aspnet-core/xml/ns-Microsoft.AspNetCore.Http.Headers.xml:3]:
```suggestion
    <summary>Contains types for getting and setting headers for HTTP requests and responses.</summary>
```

File: aspnet-core/xml/ns-Microsoft.AspNetCore.HttpsPolicy.xml [hunk 6]
```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.HttpsPolicy">
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Contains types that are used to support HTTPS Redirection and HTTP Strict-Transport-Security.</summary>
```
Comment (JamesNK) [inline aspnet-core/xml/ns-Microsoft.AspNetCore.HttpsPolicy.xml:3]:
```suggestion
    <summary>Contains types that are used to support HTTPS Redirection and HTTP Strict-Transport-Security (HSTS).</summary>
```

File: aspnet-core/xml/ns-Microsoft.AspNetCore.HttpsPolicy.xml [hunk 7]
```diff
@@ -1,6 +1,6 @@
...
     <remarks>To be added.</remarks>
```
Comment (JamesNK) [inline aspnet-core/xml/ns-Microsoft.AspNetCore.HttpsPolicy.xml:4]:
```suggestion
    <remarks>For more information, see <see href="https://docs.microsoft.com/aspnet/core/security/enforcing-ssl">Enforce HTTPS in ASP.NET Core</see>.</remarks>
```

**Review by JamesNK — APPROVED** [review]

**Review by BrennanConroy — APPROVED** [review]

**Review by Rick-Anderson — APPROVED** [review]

**Comment by wtgodbe** [comment]
@Rick-Anderson is it alright if I merge this?

**Comment by Rick-Anderson** [comment]
> @Rick-Anderson is it alright if I merge this?

Yup, that's what the green check means

image

**Pull request merged**